uaclient/messages.py

```diff
@@ -1,10 +1,21 @@
+from typing import Dict, Optional  # noqa: F401
+
 from uaclient.defaults import BASE_UA_URL, DOCUMENTATION_URL
 
 
 class NamedMessage:
     def __init__(self, name: str, msg: str):
         self.name = name
         self.msg = msg
+        self._additional_info = None  # type: Optional[Dict[str, str]]
+
+    @property
```

Comment:
As there is no operations/checks when setting/reading `additional_info`, could it be just an attribute instead of a property?

Reply:
Yeah, that one I am not entirely sure on how to proceed. I just want to be explicit about the existence of that attribute, so that's why I am using the `property` here. But maybe it is indeed unnecessary

Reply:
I'm fine with any implementation, as this will have no major impact except for the boilerplate for the property.
Maybe a sensible dosctring can point to it explicitly?
Anyway, up to you - feel free to merge it as you think it will be better